src/System.Collections.Concurrent/src/System.Collections.Concurrent.csproj

```diff
@@ -15,27 +16,32 @@
     <Compile Include="System\Collections\Concurrent\CDSCollectionETWBCLProvider.cs" />
     <Compile Include="System\Collections\Concurrent\ConcurrentBag.cs" />
     <Compile Include="System\Collections\Concurrent\ConcurrentDictionary.cs" />
-    <Compile Include="System\Collections\Concurrent\ConcurrentQueue.cs" />
     <Compile Include="System\Collections\Concurrent\ConcurrentStack.cs" />
-    <Compile Include="System\Collections\Concurrent\IProducerConsumerCollection.cs" />
     <Compile Include="System\Collections\Concurrent\OrderablePartitioner.cs" />
     <Compile Include="System\Collections\Concurrent\Partitioner.cs" />
     <Compile Include="System\Collections\Concurrent\PartitionerStatic.cs" />
     <Compile Include="System\Collections\Concurrent\PlatformHelper.cs" />
-    <Compile Include="$(CommonPath)\System\Collections\Concurrent\ConcurrentQueue_Segment.cs">
-      <Link>Common\System\Collections\Concurrent\ConcurrentQueue_Segment.cs</Link>
+    <Compile Include="$(CommonPath)\CoreLib\Internal\Padding.cs">
+      <Link>Common\CoreLib\Internal\Padding.cs</Link>
+    </Compile>
+    <Compile Include="$(CommonPath)\CoreLib\System\Collections\Concurrent\ConcurrentQueue_Segment.cs">
```

Comment:
Is ConcurrentQueue_Segment really needed here?

Reply:
yes. I removed it. Will push up the PR along with a fix for the tests